langlib/langlib-test/src/test/resources/test-src/xmllib_test.bal

```diff
@@ -154,6 +154,10 @@ function testCreateComment() returns xml {
     return 'xml:createComment("This text should be wraped in xml comment");
 }
 
+function testCreateText() returns xml {
+    return 'xml:createText("This is xml text");
```

Comment:
Shall we add a few more tests?

e.g.,
- tests for empty strings
- tests with new lines, no spaces, single character, etc.
- tests with `xml:Text` as the contextually expected type as opposed to `xml` here (e.g., assign it to an `xml:Text` variable and use it)